source/_integrations/reolink.markdown

```diff
@@ -58,6 +58,10 @@ These sensors are polled every 60 seconds and receive ONVIF push events for imme
 Not all camera models generate ONVIF push events for all event types, some binary sensors might, therefore, only be polled.
 For list of Reolink products that support ONVIF see the [Reolink Support Site](https://support.reolink.com/hc/en-us/articles/900000617826).
 
+## General notes for all entities
+
+Entities listed below that have a * behind their name are disabled by default. To enable them go to settings->devices->reolink->device->+x enties not shown, click the entity you want to enable, settings (gear) -> enable.
```

Comment:
Is it just me or if I go to the device tab I directly see a list of devices instead of the integrations that provide devices.

Reply:
I think there's probably another way to describe this or something to link to, so I think I'll let someone else who is more familiar with docs review it

Reply:
Does it now make sence to you?